articles/azure/azs-how-configure-cli.md

```diff
@@ -179,5 +179,15 @@ az vm create --resource-group testRG --name testVM --image OpenLogic:CentOS:6.9:
 There are some known issues that you must be aware of when using CLI in Azure Stack:
 
 * The CLI interactive mode i.e the `az interactive` command is not yet supported in Azure Stack.
+
 * To get the list of virtual machine images available in Azure Stack, use the `az vm images list --all` command instead of the `az vm image list` command. Specifying the `--all` option makes sure that response returns only the images that are available in your Azure Stack environment. 
+<<<<<<< HEAD
+* Virtual machine image aliases that are available in Azure may not be applicable to Azure Stack. When using virtual machine images, you must use the entire URN parameter (Canonical:UbuntuServer:14.04.3-LTS:1.0.0) instead of the image alias. This URN must match the image specifications as derived from the `az vm images list` command.
+=======
+
 * Virtual machine image aliases that are available in Azure may not be applicable to Azure Stack. When using virtual machine images, you must use the entire URN parameter (Canonical:UbuntuServer:14.04.3-LTS:1.0.0) instead of the image alias. This URN must match the image specifications as derived from the `az vm images list` command.
+
+## Feedback
+
+If you have any comments on this document or any other aspect of your UKCloud experience, send them to <products@ukcloud.com>.
+>>>>>>> 70ba515373ff46fc77c0fa4950e831aebebeffb9
```

Comment:
What's with all the >>>>>?

Reply:
These were automatically generated by github when we were merging different branches - however if you review the latest version of the document it shouldn't be present